package.json

```diff
@@ -115,9 +115,10 @@
     "@storybook/manager-webpack5": "6.3.6",
     "@storybook/react": "6.3.6",
     "@storybook/theming": "6.3.6",
-    "@testing-library/jest-dom": "5.11.9",
-    "@testing-library/react": "10.4.9",
-    "@testing-library/react-hooks": "5.0.3",
+    "@testing-library/dom": "8.11.3",
```

Comment:
why is this needed ?  I assume related to react lib major bump?

Reply:
An old version being pulled in by a nested dep was being hoisted to the root, and I wanted to make sure nothing accidentally referenced the old version.